src/porepy/utils/sort_points.py

```diff
@@ -109,6 +109,8 @@ def sort_point_plane(pts, centre, normal=None, tol=1e-5):
     map_pts: np.array, 1xn, sorted point ids.
 
     """
+    _c = np.atleast_2d(centre)
+    centre = _c if _c.shape[0] > _c.shape[1] else _c.T
```

Comment:
Will not center = center.reshape((-1, 1)) do the same, with more accessible code?

Reply:
I would think so